lisa/analysis/load_tracking.py

```diff
@@ -71,7 +81,7 @@ def _columns_to_drop(cls, event):
         """
         The extra columns not shared between trace event versions
         """
-        if event in ['sched_load_cfs_rq', 'sched_load_se', 'sched_pelt_se']:
+        if event in [*cls._SCHED_PELT_CFS_NAMES, 'sched_load_se', 'sched_pelt_se']:
```

Comment:
Same here. Why not juse _SCHED_PELT_CFS_NAMES?

Reply:
`_SCHED_PELT_CFS_NAMES` does not contain `sched_load_se`/`sched_pelt_se`